.github/workflows/test.yml

```diff
@@ -28,6 +28,12 @@ env:
   DEVELOP: ${{ (github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository) && (github.head_ref || github.ref_name) == 'develop' }}
   RELEASE: ${{ (github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository) && startsWith(github.head_ref || github.ref_name, 'release-') }}
 
+  # The current recommended version for Managed Runtime:
+  # https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/upgrade-node-version.html
+  # NOTE: if you update these values, please also update `IS_MRT_NODE`.
+  MRT_NODE: 22
+  MRT_NPM: 10
```

Comment:
Bonus: refactor the MRT node version, so that there are less places we need to update, when we bump up the node version in the future.